src/nostr-comment/nostr-comment.ts

```
@@ -0,0 +1,930 @@
+                console.log('Load operation superseded by newer request, aborting error handling');
+                return;
+            }
+
+            this.isError = true;
+            this.errorMessage = error instanceof Error ? error.message : 'Failed to load comments';
+            console.error('Error loading comments:', error);
+        } finally {
+            // Check if this load operation is still current before final state update
+            if (currentVersion !== this.loadVersion) {
+                console.log('Load operation superseded by newer request, aborting final state update');
+                return;
```

Biome (2.1.2)

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)